### PR TITLE
refactor: rename *ID to *Id

### PR DESCRIPTION
**Pull request by ayntee**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,7 +15,7 @@ dist/
 public/
 .cache/
 
-# IntelliJ IDEA
+# IntelliJ IdEA
```
Comment (itohatweb) [inline .gitignore:18]:
```suggestion
# IntelliJ IDEA
```

File: LICENSE [hunk 2]
```diff
@@ -12,7 +12,7 @@ furnished to do so, subject to the following conditions:
 The above copyright notice and this permission notice shall be included in all
 copies or substantial portions of the Software.
 
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+THE SOFTWARE IS PROVIdED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
```
Comment (itohatweb) [inline LICENSE:15]:
```suggestion
THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
```

File: src/helpers/commands/delete_slash_command.ts [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 import { RequestManager } from "../../rest/request_manager.ts";
 import { endpoints } from "../../util/constants.ts";
 
 /** Deletes a slash command. */
-export function deleteSlashCommand(id: string, guildID?: string) {
-  if (!guildID) {
-    return RequestManager.delete(endpoints.COMMANDS_ID(applicationID, id));
+export function deleteSlashCommand(id: string, guildId?: string) {
+  if (!guildId) {
+    return RequestManager.delete(endpoints.COMMANDS_Id(applicationId, id));
```
Comment (itohatweb) [inline src/helpers/commands/delete_slash_command.ts:8]:
```suggestion
    return RequestManager.delete(endpoints.COMMANDS_ID(applicationId, id));
```

File: src/helpers/commands/delete_slash_command.ts [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
   }
   return RequestManager.delete(
-    endpoints.COMMANDS_GUILD_ID(applicationID, guildID, id),
+    endpoints.COMMANDS_GUILD_Id(applicationId, guildId, id),
```
Comment (itohatweb) [inline src/helpers/commands/delete_slash_command.ts:11]:
```suggestion
    endpoints.COMMANDS_GUILD_ID(applicationId, guildId, id),
```

File: src/helpers/commands/delete_slash_response.ts [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
 
 /** To delete your response to a slash command. If a message id is not provided, it will default to deleting the original response. */
-export async function deleteSlashResponse(token: string, messageID?: string) {
+export async function deleteSlashResponse(token: string, messageId?: string) {
   const result = await RequestManager.delete(
-    messageID
-      ? endpoints.INTERACTION_ID_TOKEN_MESSAGEID(
-        applicationID,
+    messageId
+      ? endpoints.INTERACTION_Id_TOKEN_MESSAGEId(
```
Comment (itohatweb) [inline src/helpers/commands/delete_slash_response.ts:9]:
```suggestion
      ? endpoints.INTERACTION_ID_TOKEN_MESSAGEId(
```

File: src/helpers/commands/delete_slash_response.ts [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
+        applicationId,
         token,
-        messageID,
+        messageId,
       )
-      : endpoints.INTERACTION_ORIGINAL_ID_TOKEN(applicationID, token),
+      : endpoints.INTERACTION_ORIGINAL_Id_TOKEN(applicationId, token),
```
Comment (itohatweb) [inline src/helpers/commands/delete_slash_response.ts:14]:
```suggestion
      : endpoints.INTERACTION_ORIGINAL_ID_TOKEN(applicationId, token),
```

File: src/helpers/commands/edit_slash_response.ts [hunk 7]
```diff
@@ -49,14 +49,14 @@ export async function editSlashResponse(
   }
 
   const result = await RequestManager.patch(
-    options.messageID
-      ? endpoints.WEBHOOK_MESSAGE(applicationID, token, options.messageID)
-      : endpoints.INTERACTION_ORIGINAL_ID_TOKEN(applicationID, token),
+    options.messageId
+      ? endpoints.WEBHOOK_MESSAGE(applicationId, token, options.messageId)
+      : endpoints.INTERACTION_ORIGINAL_Id_TOKEN(applicationId, token),
```
Comment (itohatweb) [inline src/helpers/commands/edit_slash_response.ts:54]:
```suggestion
      : endpoints.INTERACTION_ORIGINAL_ID_TOKEN(applicationId, token),
```

File: src/helpers/commands/get_slash_command.ts [hunk 8]
```diff
@@ -1,13 +1,13 @@
...
-export async function getSlashCommand(commandID: string, guildID?: string) {
+/** Fetchs the global command for the given Id. If a guildId is provided, the guild command will be fetched. */
+export async function getSlashCommand(commandId: string, guildId?: string) {
   const result = await RequestManager.get(
-    guildID
-      ? endpoints.COMMANDS_GUILD_ID(applicationID, guildID, commandID)
-      : endpoints.COMMANDS_ID(applicationID, commandID),
+    guildId
+      ? endpoints.COMMANDS_GUILD_Id(applicationId, guildId, commandId)
+      : endpoints.COMMANDS_Id(applicationId, commandId),
```
Comment (itohatweb) [inline src/helpers/commands/get_slash_command.ts:10]:
```suggestion
      ? endpoints.COMMANDS_GUILD_ID(applicationId, guildId, commandId)
      : endpoints.COMMANDS_ID(applicationId, commandId),
```

File: src/helpers/commands/send_interaction_response.ts [hunk 9]
```diff
@@ -39,7 +39,7 @@ export async function sendInteractionResponse(
   }
 
   const result = await RequestManager.post(
-    endpoints.INTERACTION_ID_TOKEN(id, token),
+    endpoints.INTERACTION_Id_TOKEN(id, token),
```
Comment (itohatweb) [inline src/helpers/commands/send_interaction_response.ts:42]:
```suggestion
    endpoints.INTERACTION_ID_TOKEN(id, token),
```

File: src/helpers/commands/upsert_slash_command.ts [hunk 10]
```diff
@@ -7,16 +7,16 @@ import { validateSlashCommands } from "../../util/utils.ts";
...
 ) {
   validateSlashCommands([options]);
 
   const result = await RequestManager.patch(
-    guildID
-      ? endpoints.COMMANDS_GUILD_ID(applicationID, guildID, commandID)
-      : endpoints.COMMANDS_ID(applicationID, commandID),
+    guildId
+      ? endpoints.COMMANDS_GUILD_Id(applicationId, guildId, commandId)
+      : endpoints.COMMANDS_Id(applicationId, commandId),
```
Comment (itohatweb) [inline src/helpers/commands/upsert_slash_command.ts:19]:
```suggestion
      ? endpoints.COMMANDS_GUILD_ID(applicationId, guildId, commandId)
      : endpoints.COMMANDS_ID(applicationId, commandId),
```

File: src/helpers/guilds/edit_widget.ts [hunk 11]
```diff
@@ -4,15 +4,15 @@ import { requireBotGuildPermissions } from "../../util/permissions.ts";
...
+  guildId: string,
   enabled: boolean,
-  channelID?: string | null,
+  channelId?: string | null,
 ) {
-  await requireBotGuildPermissions(guildID, ["MANAGE_GUILD"]);
+  await requireBotGuildPermissions(guildId, ["MANAGE_GUILD"]);
 
-  const result = await RequestManager.patch(endpoints.GUILD_WIDGET(guildID), {
+  const result = await RequestManager.patch(endpoints.GUILD_WIdGET(guildId), {
```
Comment (itohatweb) [inline src/helpers/guilds/edit_widget.ts:13]:
```suggestion
  const result = await RequestManager.patch(endpoints.GUILD_WIDGET(guildId), {
```

File: src/helpers/guilds/get_widget.ts [hunk 12]
```diff
@@ -3,12 +3,12 @@ import { RequestManager } from "../../rest/request_manager.ts";
...
 
 /** Returns the widget for the guild. */
-export async function getWidget(guildID: string, options?: { force: boolean }) {
+export async function getWidget(guildId: string, options?: { force: boolean }) {
   if (!options?.force) {
-    const guild = await cacheHandlers.get("guilds", guildID);
+    const guild = await cacheHandlers.get("guilds", guildId);
     if (!guild) throw new Error(Errors.GUILD_NOT_FOUND);
-    if (!guild?.widgetEnabled) throw new Error(Errors.GUILD_WIDGET_NOT_ENABLED);
+    if (!guild?.widgetEnabled) throw new Error(Errors.GUILD_WIdGET_NOT_ENABLED);
```
Comment (itohatweb) [inline src/helpers/guilds/get_widget.ts:10]:
```suggestion
    if (!guild?.widgetEnabled) throw new Error(Errors.GUILD_WIDGET_NOT_ENABLED);
```

File: src/helpers/guilds/get_widget.ts [hunk 13]
```diff
@@ -3,12 +3,12 @@ import { RequestManager } from "../../rest/request_manager.ts";
...
   }
 
-  return RequestManager.get(`${endpoints.GUILD_WIDGET(guildID)}.json`);
+  return RequestManager.get(`${endpoints.GUILD_WIdGET(guildId)}.json`);
```
Comment (itohatweb) [inline src/helpers/guilds/get_widget.ts:13]:
```suggestion
  return RequestManager.get(`${endpoints.GUILD_WIDGET(guildId)}.json`);
```

File: src/helpers/guilds/get_widget_image_url.ts [hunk 14]
```diff
@@ -3,18 +3,18 @@ import { endpoints } from "../../util/constants.ts";
...
     style?: "shield" | "banner1" | "banner2" | "banner3" | "banner4";
     force?: boolean;
   },
 ) {
   if (!options?.force) {
-    const guild = await cacheHandlers.get("guilds", guildID);
+    const guild = await cacheHandlers.get("guilds", guildId);
     if (!guild) throw new Error(Errors.GUILD_NOT_FOUND);
-    if (!guild.widgetEnabled) throw new Error(Errors.GUILD_WIDGET_NOT_ENABLED);
+    if (!guild.widgetEnabled) throw new Error(Errors.GUILD_WIdGET_NOT_ENABLED);
```
Comment (itohatweb) [inline src/helpers/guilds/get_widget_image_url.ts:15]:
```suggestion
    if (!guild.widgetEnabled) throw new Error(Errors.GUILD_WIDGET_NOT_ENABLED);
```

File: src/helpers/guilds/get_widget_image_url.ts [hunk 15]
```diff
@@ -3,18 +3,18 @@ import { endpoints } from "../../util/constants.ts";
...
   }
 
-  return `${endpoints.GUILD_WIDGET(guildID)}.png?style=${options?.style ??
+  return `${endpoints.GUILD_WIdGET(guildId)}.png?style=${options?.style ??
```
Comment (itohatweb) [inline src/helpers/guilds/get_widget_image_url.ts:18]:
```suggestion
  return `${endpoints.GUILD_WIDGET(guildId)}.png?style=${options?.style ??
```

File: src/helpers/guilds/get_widget_settings.ts [hunk 16]
```diff
@@ -3,10 +3,10 @@ import { endpoints } from "../../util/constants.ts";
 import { requireBotGuildPermissions } from "../../util/permissions.ts";
 
 /** Returns the guild widget object. Requires the MANAGE_GUILD permission. */
-export async function getWidgetSettings(guildID: string) {
-  await requireBotGuildPermissions(guildID, ["MANAGE_GUILD"]);
+export async function getWidgetSettings(guildId: string) {
+  await requireBotGuildPermissions(guildId, ["MANAGE_GUILD"]);
 
-  const result = await RequestManager.get(endpoints.GUILD_WIDGET(guildID));
+  const result = await RequestManager.get(endpoints.GUILD_WIdGET(guildId));
```
Comment (itohatweb) [inline src/helpers/guilds/get_widget_settings.ts:9]:
```suggestion
  const result = await RequestManager.get(endpoints.GUILD_WIDGET(guildId));
```

File: src/helpers/members/edit_bot_profile.ts [hunk 17]
```diff
@@ -17,10 +17,10 @@ export async function editBotProfile(username?: string, botAvatarURL?: string) {
       throw new Error(Errors.USERNAME_MIN_LENGTH);
     }
     if (["@", "#", ":", "```"].some((char) => username.includes(char))) {
-      throw new Error(Errors.USERNAME_INVALID_CHARACTER);
+      throw new Error(Errors.USERNAME_INVALId_CHARACTER);
```
Comment (itohatweb) [inline src/helpers/members/edit_bot_profile.ts:20]:
```suggestion
      throw new Error(Errors.USERNAME_INVALID_CHARACTER);
```

File: src/helpers/members/edit_bot_profile.ts [hunk 18]
```diff
@@ -17,10 +17,10 @@ export async function editBotProfile(username?: string, botAvatarURL?: string) {
...
     }
     if (["discordtag", "everyone", "here"].includes(username)) {
-      throw new Error(Errors.USERNAME_INVALID_USERNAME);
+      throw new Error(Errors.USERNAME_INVALId_USERNAME);
```
Comment (itohatweb) [inline src/helpers/members/edit_bot_profile.ts:23]:
```suggestion
      throw new Error(Errors.USERNAME_INVALID_USERNAME);
```

File: src/helpers/webhooks/create_webhook.ts [hunk 19]
```diff
@@ -21,11 +21,11 @@ export async function createWebhook(
     [...options.name].length < 2 ||
     [...options.name].length > 32
   ) {
-    throw new Error(Errors.INVALID_WEBHOOK_NAME);
+    throw new Error(Errors.INVALId_WEBHOOK_NAME);
```
Comment (itohatweb) [inline src/helpers/webhooks/create_webhook.ts:24]:
```suggestion
    throw new Error(Errors.INVALID_WEBHOOK_NAME);
```

File: src/helpers/webhooks/delete_webhook.ts [hunk 20]
```diff
@@ -3,10 +3,10 @@ import { endpoints } from "../../util/constants.ts";
 import { requireBotChannelPermissions } from "../../util/permissions.ts";
 
 /** Delete a webhook permanently. Requires the `MANAGE_WEBHOOKS` permission. Returns a undefined on success */
-export async function deleteWebhook(channelID: string, webhookID: string) {
-  await requireBotChannelPermissions(channelID, ["MANAGE_WEBHOOKS"]);
+export async function deleteWebhook(channelId: string, webhookId: string) {
+  await requireBotChannelPermissions(channelId, ["MANAGE_WEBHOOKS"]);
 
-  const result = await RequestManager.delete(endpoints.WEBHOOK_ID(webhookID));
+  const result = await RequestManager.delete(endpoints.WEBHOOK_Id(webhookId));
```
Comment (itohatweb) [inline src/helpers/webhooks/delete_webhook.ts:9]:
```suggestion
  const result = await RequestManager.delete(endpoints.WEBHOOK_ID(webhookId));
```

File: src/helpers/webhooks/edit_webhook.ts [hunk 21]
```diff
@@ -4,15 +4,15 @@ import { requireBotChannelPermissions } from "../../util/permissions.ts";
...
-  webhookID: string,
+  channelId: string,
+  webhookId: string,
   options: WebhookEditOptions,
 ) {
-  await requireBotChannelPermissions(channelID, ["MANAGE_WEBHOOKS"]);
+  await requireBotChannelPermissions(channelId, ["MANAGE_WEBHOOKS"]);
 
-  const result = await RequestManager.patch(endpoints.WEBHOOK_ID(webhookID), {
+  const result = await RequestManager.patch(endpoints.WEBHOOK_Id(webhookId), {
```
Comment (itohatweb) [inline src/helpers/webhooks/edit_webhook.ts:13]:
```suggestion
  const result = await RequestManager.patch(endpoints.WEBHOOK_ID(webhookId), {
```

File: src/helpers/webhooks/execute_webhook.ts [hunk 22]
```diff
@@ -2,14 +2,14 @@ import { RequestManager } from "../../rest/request_manager.ts";
...
+/** Execute a webhook with webhook Id and webhook token */
 export async function executeWebhook(
-  webhookID: string,
+  webhookId: string,
   webhookToken: string,
   options: ExecuteWebhookOptions,
 ) {
   if (!options.content && !options.file && !options.embeds) {
-    throw new Error(Errors.INVALID_WEBHOOK_OPTIONS);
+    throw new Error(Errors.INVALId_WEBHOOK_OPTIONS);
```
Comment (itohatweb) [inline src/helpers/webhooks/execute_webhook.ts:12]:
```suggestion
    throw new Error(Errors.INVALID_WEBHOOK_OPTIONS);
```

File: src/helpers/webhooks/get_webhook.ts [hunk 23]
```diff
@@ -2,8 +2,8 @@ import { RequestManager } from "../../rest/request_manager.ts";
 import { endpoints } from "../../util/constants.ts";
 
 /** Returns the new webhook object for the given id. */
-export async function getWebhook(webhookID: string) {
-  const result = await RequestManager.get(endpoints.WEBHOOK_ID(webhookID));
+export async function getWebhook(webhookId: string) {
+  const result = await RequestManager.get(endpoints.WEBHOOK_Id(webhookId));
```
Comment (itohatweb) [inline src/helpers/webhooks/get_webhook.ts:6]:
```suggestion
  const result = await RequestManager.get(endpoints.WEBHOOK_ID(webhookId));
```

File: src/rest/cache.ts [hunk 24]
```diff
@@ -4,7 +4,7 @@ export const restCache: RestCache = {
   globallyRateLimited: false,
   ratelimitedPaths: new Map(),
   eventHandlers: {
-    // BY DEFAULT WE WILL LOG ALL ERRORS TO CONSOLE. USER CAN CHOOSE TO OVERRIDE
+    // BY DEFAULT WE WILL LOG ALL ERRORS TO CONSOLE. USER CAN CHOOSE TO OVERRIdE
```
Comment (itohatweb) [inline src/rest/cache.ts:7]:
```suggestion
    // BY DEFAULT WE WILL LOG ALL ERRORS TO CONSOLE. USER CAN CHOOSE TO OVERRIDE
```

File: src/rest/request.ts [hunk 25]
```diff
@@ -49,7 +49,7 @@ export function createRequestBody(queuedRequest: QueuedRequest) {
     queuedRequest.payload.body = undefined;
   }
 
-  // IF A REASON IS PROVIDED ENCODE IT IN HEADERS
+  // IF A REASON IS PROVIdED ENCODE IT IN HEADERS
```
Comment (itohatweb) [inline src/rest/request.ts:52]:
```suggestion
  // IF A REASON IS PROVIDED ENCODE IT IN HEADERS
```

File: src/util/constants.ts [hunk 26]
```diff
@@ -32,145 +32,145 @@ export const endpoints = {
...
-  GUILD_VANITY_URL: (guildID: string) => `${GUILDS_BASE(guildID)}/vanity-url`,
-  GUILD_WEBHOOKS: (guildID: string) => `${GUILDS_BASE(guildID)}/webhooks`,
+  GUILD_AUDIT_LOGS: (guildId: string) => `${GUILDS_BASE(guildId)}/audit-logs`,
+  GUILD_BAN: (guildId: string, userId: string) =>
+    `${GUILDS_BASE(guildId)}/bans/${userId}`,
+  GUILD_BANS: (guildId: string) => `${GUILDS_BASE(guildId)}/bans`,
+  GUILD_BANNER: (guildId: string, icon: string) =>
+    `${baseEndpoints.CDN_URL}/banners/${guildId}/${icon}`,
+  GUILD_CHANNELS: (guildId: string) => `${GUILDS_BASE(guildId)}/channels`,
+  GUILD_WIdGET: (guildId: string) => `${GUILDS_BASE(guildId)}/widget`,
```
Comment (itohatweb) [inline src/util/constants.ts:86]:
```suggestion
  GUILD_WIDGET: (guildId: string) => `${GUILDS_BASE(guildId)}/widget`,
```

File: src/util/constants.ts [hunk 27]
```diff
@@ -32,145 +32,145 @@ export const endpoints = {
...
-    `${baseEndpoints.BASE_URL}/webhooks/${webhookID}`,
-  WEBHOOK_MESSAGE: (webhookID: string, token: string, messageID: string) =>
-    `${baseEndpoints.BASE_URL}/webhooks/${webhookID}/${token}/messages/${messageID}`,
-  WEBHOOK_SLACK: (webhookID: string, token: string) =>
-    `${baseEndpoints.BASE_URL}/webhooks/${webhookID}/${token}/slack`,
-  WEBHOOK_GITHUB: (webhookID: string, token: string) =>
-    `${baseEndpoints.BASE_URL}/webhooks/${webhookID}/${token}/github`,
+  WEBHOOK: (webhookId: string, token: string) =>
+    `${baseEndpoints.BASE_URL}/webhooks/${webhookId}/${token}`,
+  WEBHOOK_Id: (webhookId: string) =>
```
Comment (itohatweb) [inline src/util/constants.ts:128]:
```suggestion
  WEBHOOK_ID: (webhookId: string) =>
```

File: src/util/constants.ts [hunk 28]
```diff
@@ -32,145 +32,145 @@ export const endpoints = {
...
-    `${baseEndpoints.BASE_URL}/applications/${applicationID}/commands/${commandID}`,
-  COMMANDS_GUILD_ID: (
-    applicationID: string,
-    guildID: string,
-    commandID: string,
+  COMMANDS: (applicationId: string) =>
+    `${baseEndpoints.BASE_URL}/applications/${applicationId}/commands`,
+  COMMANDS_GUILD: (applicationId: string, guildId: string) =>
+    `${baseEndpoints.BASE_URL}/applications/${applicationId}/guilds/${guildId}/commands`,
+  COMMANDS_Id: (applicationId: string, commandId: string) =>
```
Comment (itohatweb) [inline src/util/constants.ts:142]:
```suggestion
  COMMANDS_ID: (applicationId: string, commandId: string) =>
```

File: src/util/constants.ts [hunk 29]
```diff
@@ -32,145 +32,145 @@ export const endpoints = {
...
+    `${baseEndpoints.BASE_URL}/applications/${applicationId}/commands/${commandId}`,
+  COMMANDS_GUILD_Id: (
```
Comment (itohatweb) [inline src/util/constants.ts:144]:
```suggestion
  COMMANDS_GUILD_ID: (
```

File: src/util/constants.ts [hunk 30]
```diff
@@ -32,145 +32,145 @@ export const endpoints = {
...
+    `${baseEndpoints.BASE_URL}/applications/${applicationId}/guilds/${guildId}/commands/${commandId}`,
 
   // Interaction Endpoints
-  INTERACTION_ID_TOKEN: (interactionID: string, token: string) =>
-    `${baseEndpoints.BASE_URL}/interactions/${interactionID}/${token}/callback`,
-  INTERACTION_ORIGINAL_ID_TOKEN: (interactionID: string, token: string) =>
-    `${baseEndpoints.BASE_URL}/webhooks/${interactionID}/${token}/messages/@original`,
-  INTERACTION_ID_TOKEN_MESSAGEID: (
-    applicationID: string,
+  INTERACTION_Id_TOKEN: (interactionId: string, token: string) =>
```
Comment (itohatweb) [inline src/util/constants.ts:152]:
```suggestion
  INTERACTION_ID_TOKEN: (interactionId: string, token: string) =>
```

File: src/util/constants.ts [hunk 31]
```diff
@@ -32,145 +32,145 @@ export const endpoints = {
...
+    `${baseEndpoints.BASE_URL}/interactions/${interactionId}/${token}/callback`,
+  INTERACTION_ORIGINAL_Id_TOKEN: (interactionId: string, token: string) =>
```
Comment (itohatweb) [inline src/util/constants.ts:154]:
```suggestion
  INTERACTION_ORIGINAL_ID_TOKEN: (interactionId: string, token: string) =>
```

File: src/util/constants.ts [hunk 32]
```diff
@@ -32,145 +32,145 @@ export const endpoints = {
...
+    `${baseEndpoints.BASE_URL}/webhooks/${interactionId}/${token}/messages/@original`,
+  INTERACTION_Id_TOKEN_MESSAGEId: (
```
Comment (itohatweb) [inline src/util/constants.ts:156]:
```suggestion
  INTERACTION_ID_TOKEN_MESSAGEId: (
```

File: src/util/utils.ts [hunk 33]
```diff
@@ -52,16 +52,16 @@ export const formatImageURL = (
 };
 
 function camelToSnakeCase(text: string) {
-  return text.replace(/ID|[A-Z]/g, ($1) => {
-    if ($1 === "ID") return "_id";
+  return text.replace(/Id|[A-Z]/g, ($1) => {
+    if ($1 === "Id") return "_id";
```
Comment (itohatweb) [inline src/util/utils.ts:56]: Remove this entirely

File: src/util/utils.ts [hunk 34]
```diff
@@ -52,16 +52,16 @@ export const formatImageURL = (
...
 
     return `_${$1.toLowerCase()}`;
   });
 }
 
 function snakeToCamelCase(text: string) {
   return text.replace(/_id|([-_][a-z])/ig, ($1) => {
-    if ($1 === "_id") return "ID";
+    if ($1 === "_id") return "Id";
```
Comment (itohatweb) [inline src/util/utils.ts:64]: Remove this entirely

File: src/util/utils.ts [hunk 35]
```diff
@@ -121,7 +121,7 @@ function validateSlashOptionChoices(
 ) {
   for (const choice of choices) {
     if ([...choice.name].length < 1 || [...choice.name].length > 100) {
-      throw new Error(Errors.INVALID_SLASH_OPTIONS_CHOICES);
+      throw new Error(Errors.INVALId_SLASH_OPTIONS_CHOICES);
```
Comment (itohatweb) [inline src/util/utils.ts:124]:
```suggestion
      throw new Error(Errors.INVALID_SLASH_OPTIONS_CHOICES);
```

File: src/util/utils.ts [hunk 36]
```diff
@@ -131,7 +131,7 @@ function validateSlashOptionChoices(
       (optionType === SlashCommandOptionType.INTEGER &&
         typeof choice.value !== "number")
     ) {
-      throw new Error(Errors.INVALID_SLASH_OPTIONS_CHOICES);
+      throw new Error(Errors.INVALId_SLASH_OPTIONS_CHOICES);
```
Comment (itohatweb) [inline src/util/utils.ts:134]:
```suggestion
      throw new Error(Errors.INVALID_SLASH_OPTIONS_CHOICES);
```

File: src/util/utils.ts [hunk 37]
```diff
@@ -145,15 +145,15 @@ function validateSlashOptions(options: SlashCommandOption[]) {
         (option.type !== SlashCommandOptionType.STRING &&
           option.type !== SlashCommandOptionType.INTEGER))
     ) {
-      throw new Error(Errors.INVALID_SLASH_OPTIONS_CHOICES);
+      throw new Error(Errors.INVALId_SLASH_OPTIONS_CHOICES);
```
Comment (itohatweb) [inline src/util/utils.ts:148]:
```suggestion
      throw new Error(Errors.INVALID_SLASH_OPTIONS_CHOICES);
```

File: src/util/utils.ts [hunk 38]
```diff
@@ -145,15 +145,15 @@ function validateSlashOptions(options: SlashCommandOption[]) {
...
     }
 
     if (
       ([...option.name].length < 1 || [...option.name].length > 32) ||
       ([...option.description].length < 1 ||
         [...option.description].length > 100)
     ) {
-      throw new Error(Errors.INVALID_SLASH_OPTIONS_CHOICES);
+      throw new Error(Errors.INVALId_SLASH_OPTIONS_CHOICES);
```
Comment (itohatweb) [inline src/util/utils.ts:156]:
```suggestion
      throw new Error(Errors.INVALID_SLASH_OPTIONS_CHOICES);
```

File: src/util/utils.ts [hunk 39]
```diff
@@ -172,7 +172,7 @@ export function validateSlashCommands(
       (command.name && !SLASH_COMMANDS_NAME_REGEX.test(command.name)) ||
       (create && !command.name)
     ) {
-      throw new Error(Errors.INVALID_SLASH_NAME);
+      throw new Error(Errors.INVALId_SLASH_NAME);
```
Comment (itohatweb) [inline src/util/utils.ts:175]:
```suggestion
      throw new Error(Errors.INVALID_SLASH_NAME);
```

File: src/util/utils.ts [hunk 40]
```diff
@@ -181,12 +181,12 @@ export function validateSlashCommands(
           [...command.description].length > 100)) ||
       (create && !command.description)
     ) {
-      throw new Error(Errors.INVALID_SLASH_DESCRIPTION);
+      throw new Error(Errors.INVALId_SLASH_DESCRIPTION);
```
Comment (itohatweb) [inline src/util/utils.ts:184]:
```suggestion
      throw new Error(Errors.INVALID_SLASH_DESCRIPTION);
```

File: src/util/utils.ts [hunk 41]
```diff
@@ -181,12 +181,12 @@ export function validateSlashCommands(
...
     }
 
     if (command.options?.length) {
       if (command.options.length > 25) {
-        throw new Error(Errors.INVALID_SLASH_OPTIONS);
+        throw new Error(Errors.INVALId_SLASH_OPTIONS);
```
Comment (itohatweb) [inline src/util/utils.ts:189]:
```suggestion
        throw new Error(Errors.INVALID_SLASH_OPTIONS);
```

**Review by itohatweb — CHANGES_REQUESTED** [review]
What was so bad about my PR?

File: src/helpers/commands/delete_slash_command.ts (hunk 4 above)
Comment (ayntee) [reply src/helpers/commands/delete_slash_command.ts:11]: Thank you

File: src/helpers/commands/delete_slash_response.ts (hunk 5 above)
Comment (ayntee) [reply src/helpers/commands/delete_slash_response.ts:9]: Is this correct?

Comment (itohatweb) [reply src/helpers/commands/delete_slash_response.ts:9]: I didn't saw the second Id

Comment (ayntee) [reply src/helpers/commands/delete_slash_response.ts:9]: Isn't there supposed be an underscore after `MESSAGE`?
`INTERACTION_ID_TOKEN_MESSAGEID` -> `INTERACTION_ID_TOKEN_MESSAGE_ID`

Comment (itohatweb) [reply src/helpers/commands/delete_slash_response.ts:9]: yeah you're right

File: src/util/constants.ts (hunk 32 above)
Comment (ayntee) [reply src/util/constants.ts:156]: Is this correct?

File: src/util/utils.ts (hunk 33 above)
Comment (ayntee) [reply src/util/utils.ts:56]: Right

File: src/util/utils.ts (hunk 34 above)
Comment (ayntee) [reply src/util/utils.ts:64]: :+1:

File: src/util/constants.ts (hunk 32 above)
Comment (itohatweb) [reply src/util/constants.ts:156]: yeah...

**Pull request merged**